Packages/MIES/MIES_DAEphys.ipf

```diff
@@ -1347,6 +1343,10 @@ Function DAP_OneTimeCallAfterDAQ(panelTitle, [forcedStop, startTPAfterDAQ])
 		TP_TeardownCommon(panelTitle)
 	endif
 
+	if(!forcedStop)
+		AFM_CallAnalysisFunctions(panelTitle, POST_DAQ_EVENT)
+	endif
+
 	DAP_ApplyDelayedClampModeChange(panelTitle)
```

Comment:
@timjarsky Can we move it even further after `DAP_ApplyDelayedClampModeChange(panelTitle)`?

Reply:
@timjarsky Ditch that. This does not make sense.